scripts/docker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM microsoft/dotnet:2.0.3-runtime-jessie
+FROM microsoft/dotnet:2.1-aspnetcore-runtime
```

Comment:
Please change the maintainer to our team's alias

Reply:
updated